nodejs/src/handlers/mint_badges.ts

```diff
@@ -1,14 +1,14 @@
 
 /** 
-  * Mint badges for a user.
+  * Mint a badge for a user.
```

Comment:
Rename the file to `mint_badge` now that it's for a single badge

Reply:
True. Will fix.